src/screens/DetailDomain/DetailDomainScreen.js

```diff
@@ -343,7 +342,7 @@ const styles = StyleSheet.create({
       height: 1
     },
     shadowOpacity: 0.5,
-    backgroundColor: COLORS.white
+    backgroundColor: 'white'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hardcoded color value 'white' should be replaced with a constant for better maintainability and consistency across the application. If the color needs to be changed in the future, it would be easier to change it in one place (the constant) rather than searching through the entire codebase.
```diff
-    backgroundColor: 'white'
+    backgroundColor: colors.white,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->